### DISCUSSION TITLE
Uso de atributos no Command

### DISCUSSION THREAD

**Discussion by RanieryCG** [balta-io/7196#15] [Perguntas e Respostas] [answered]
Olá, tudo bem? 

Foi criado command para receber os request e respostas e usado o Flunt para fazer a validação.

Seria errado usar nos atributos do command os decoradores para validar?

Exemplo, Required, MaxLength e etc?

Não seria uma forma nativa de fazer isso? Está fora do padrão do mercado?

Desde já agradeço as respostas, abraço!

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @RanieryCG seria uma outra abordagem sim... não está fora de mercado nem é errado, é apenas uma outra abordagem.
Se você se identifica mais com ela, pode usar tranquilamente :)
